mastermindview/src/views/Home.vue

```diff
@@ -70,22 +77,30 @@ export default {
       this.Row = response;
       console.log(response);
       var Row = this.$children[0].$children.find(child => {return child.RowId == this.currentRow});
+      
       if(this.Row.clues[0] != 'BLANK') {
         Row.$children[4].Color = this.Row.clues[0];
       }
       if(this.Row.clues[1] != 'BLANK') {
-        Row.$children[5].Color = this.Row.clues[1]; 
+        Row.$children[5].Color = this.Row.clues[1];
       }
       if(this.Row.clues[2] != 'BLANK') {
         Row.$children[6].Color = this.Row.clues[2];
       }
       if(this.Row.clues[3] != 'BLANK') {
         Row.$children[7].Color = this.Row.clues[3];
       }
+      if(this.Row.clues[0] == 'BLACK' && this.Row.clues[1] == 'BLACK' && this.Row.clues[2] == 'BLACK' && this.Row.clues[3] == 'BLACK') {
+        this.WinGame();
+      }
+      this.setNextRow();
       if(this.Row.clues[0] != null){
         this.setNextRow();
       }
     },
+    WinGame() {
+      this.showWinMessage();
+    },
```

Comment:
Wordt deze ook aangeroepen vanuit this.setNextRow()?

Reply:
nee alleen this.LostGame() wordt aangeroepen in this.setNextRow(). WinGame() wordt aangeroepen wanneer bij changeclues er 4 zwarte clues ontstaan

Reply:
prima